src/McBopomofo.cpp

```diff
@@ -238,6 +259,16 @@ void McBopomofoEngine::activate(const fcitx::InputMethodEntry&,
   keyHandler_->setPutLowercaseLettersToComposingBuffer(
       config_.shiftLetterKeys.value() == ShiftLetterKeys::PutLowercaseToBuffer);
 
+  if (config_.ctrlEnterKeys.value() == CtrlEnterKey::Disabled) {
```

Comment:
Nit: this is error-prone here when a new enum value is added in the future… want to use `switch` instead so that at least the default value is always set to `Disabled`?